data/data/azure/bootstrap/main.tf

```diff
@@ -0,0 +1,156 @@
+
+resource "random_string" "storage_suffix" {
+  length = 5
+  upper = false
+  special = false
+  keepers = {
+        # Generate a new ID only when a new resource group is defined
+        resource_group = "${var.resource_group_name}"
+    }
+}
+
+resource "azurerm_storage_account" "ignition" {
+  name = "ignitiondata${random_string.storage_suffix.result}"
```

Comment:
question: why use a random string here.?
in aws we keep our resources unique and per cluster by using [`cluster_id`](https://github.com/openshift/installer/blob/b3b8a8928b1953c00944fae307c933dbcad6cf75/data/data/config.tf#L66-L75)

Reply:
The storage account name has strict requirements and limitations. It needs to be globally unique on Azure and it cannot include special characters or spaces. Generating a random suffix is safer.
The other option is to generate it in go and add it in the azure tfvars.

Reply:
> It needs to be globally unique on Azure
oh, maybe then it is fine :)